code/modules/vending/wardrobes.dm

```diff
@@ -63,7 +64,8 @@
 	product_ads = "Guaranteed to protect your feet from industrial accidents!;Afraid of radiation? Then wear yellow!"
 	vend_reply = "Thank you for using the EngiDrobe!"
 	products = list(/obj/item/clothing/accessory/pocketprotector = 5,
-					/obj/item/storage/backpack/duffelbag/engineering = 3,
+					/obj/item/clothing/head/beret/eng = 3,
+					/obj/item/storage/backpack/duffelbag/engineering = 2,
```

Comment:
Why is this lowered?

Reply:
Had moved to put beret at top so bags are still next to each other on the menu

Reply:
I think they mean why did you lower the amount of bags

Reply:
Oh, accidental. Will have to fix.